### ISSUE TITLE
Feature request - Quick edit message

### ISSUE THREAD

**Issue by xmedeko** [RocketChat/feature-requests#644]
## My Setup
- Operating System: Windows
- App Version: 2.15.5
- Installation type: Electron Windows app system wide

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
I do edit my messages quite frequently, e.g. fixing typos etc. I would like to have a possibility do edit (delete) messages quickly.

### Current Behavior
I have to click three dots in the right and choose a proper action form the long menu. The only quick action is to flag with emoji. 

### Expected Behavior
Add icons to edit and delete the message, below the "emoji" icon when I hover my message.
(E.g. Similar behaviour has Google Hangouts Chat). 

Also, **in place** editing would be very very nice.

Optional: a double click to edit would be cool as well. Then the deleting message could be accomplished by double click, delete all text.

**Comment by johnyb0y** [comment]
Just press the arrow key on your keyboard. You can navigate your last messages and edit without clicking anything.

**Comment by xmedeko** [comment]
Well, that's not so intuitive, but works, thanks. (Alt+arrows make listing faster in case of multiline messages.)

Anyway, I vote for in place editing. Feel free to close this issue if you do not plan in place editing.

**Comment by nemeth-it** [comment]
Yes, missing this feature, too. "Double click" or "Option/Alt+Click" on a posting, to enter just an inline-editing textbox.

**Comment by jeanfbrito** [comment]
Please test if the latest release did solve this. If not please reopen the issue or create a new one and tell us. I will be closing this one for now.

Thank you!

**Comment by xmedeko** [comment]
@jeanfbrito no, even with 3.7.5 I do not see any in place message editing. Maybe this FR belong to https://github.com/RocketChat/Rocket.Chat repository?

Note: I cannot reopen the issue since I am not a collaborator.

**Comment by jeanfbrito** [comment]
Oh sorry, I misunderstood the issue. Yes, a FR on the Rocket.Chat repo would be interesting because this is a chat feature directly.
I will move this to there.
